rdr_client/hpo_assigner.py

```diff
@@ -13,31 +13,33 @@
 
 import csv
 import logging
-
 from main_util import get_parser, configure_logging
 
 from client import Client, HttpException, client_log
 
-_ORG_PREFIX = 'Organization/'
-
 
 def main(client):
   num_no_change = 0
   num_updates = 0
   num_errors = 0
+
   with open(client.args.file) as csvfile:
     reader = csv.reader(csvfile)
     for line in reader:
       try:
-        participant_id, hpo = [v.strip() for v in line]
+        participant_id, new_pairing = [v.strip() for v in line]
+        pairing = client.args.pairing
+        assert pairing in ('site', 'organization', 'awardee')
```

Comment:
We should do this check once, up front, and bail out if it fails... not on every line.